incubator/node/README.md

```diff
@@ -83,3 +96,98 @@ The [Bitnami Node](https://github.com/bitnami/bitnami-docker-node) image stores
 
 Persistent Volume Claims are used to keep the data across deployments. This is known to work in GCE, AWS, and minikube.
 See the [Configuration](#configuration) section to configure the PVC or to disable persistence.
+
+## Set an Ingress controller
+
+First install the nginx-ingress controller via helm:
+
+```
+$ helm install stable/nginx-ingress
+```
+
+Now deploy the node helm chart:
+
+```
+$ helm install --name my-release bitnami-incubator/node --set ingress.enabled=true,ingress.host=example.com,serviceType=ClusterIP
+```
+
+### Configure TLS termination for your ingress controller
+
+You must manually create a secret containing the certificate and key for your domain. You can do it with this command:
+
+```
+$ kubectl create secret tls my-tls-secret --cert=path/to/file.cert --key=path/to/file.key
+```
+
+Then ensure you deploy the Helm chart with the following ingress configuration:
+
+```
+ingress:
+  enabled: false
+  path: /
+  host: example.com
+  annotations:
+    kubernetes.io/ingress.class: nginx
+  tls:
+      hosts:
+        - example.com
+```
+
+## Connect your application to an already existing database
+
+1. Create a secret containing your database credentials:
+
+  ```
+  $ kubectl create secret generic my-database-secret --from-literal=host=YOUR_DATABASE_HOST --from-literal=port=YOUR_DATABASE_PORT --from-literal=user=YOUR_DATABASE_USER --from-literal=password=YOUR_DATABASE_PASSWORD
+  ```
+  
+  `YOUR_DATABASE_HOST`, `YOUR_DATABASE_PORT`, `YOUR_DATABASE_USER` and `YOUR_DATABASE_PASSWORD` are placeholders that must be replaced with correct values.
+
+2. Deploy the node chart specifying the secret name
+
+  ```
+  $ helm install --name node-app --set mongodb.install=false,externaldb.secretName=my-database-secret bitnami-incubator/node
+  ```
+
+## Provision a database using the Open Service Broker for Azure
+
+1. Install Service Catalog in your Kubernetes cluster following [this instructions](https://kubernetes.io/docs/tasks/service-catalog/install-service-catalog-using-helm/)
+2. Install the Open Service Broker for Azure in your Kubernetes cluster following [this instructions](https://github.com/Azure/helm-charts/tree/master/open-service-broker-azure)
+3. Create and deploy a ServiceBinding to provision a database server in Azure cloud.
+
+  ```
+  apiVersion: servicecatalog.k8s.io/v1beta1
+  kind: ServiceInstance
+  metadata:
+    name: azure-mongodb-instance
+    labels:
+      app: mongodb
+  spec:
+    clusterServiceClassExternalName: azure-cosmos-mongo-db
+    clusterServicePlanExternalName: mongo-db
+    parameters:
+      location: YOUR_AZURE_LOCATION
+      resourceGroup: mongodb-k8s-service-catalog
+      sslEnforcement: disabled
+      firewallStartIPAddress: 0.0.0.0
+      firewallEndIPAddress: 255.255.255.255
```

Comment:
the firewall specification has changed for osba, see https://github.com/Azure/helm-charts/commit/75ab2232e80c1469ad262faa8039ac64262c5c8c#diff-d04a73ad2755189100f39bf28c84a22e for the necessary changes.

Reply:
thanks!